src/pages/Home.jsx

```diff
@@ -0,0 +1,22 @@
+import { Nav, ScrollUp } from '../components/components';
+import { Header, Main, Footer } from '../layouts/layouts';
+import { HomeIntro, NewArrivals, Testimonials } from '../sections/sections';
+
+const Home = () => {
+  return (
+    <>
+      <Header>
+        <Nav />
+      </Header>
+      <Main>
+        <HomeIntro />
+        <NewArrivals />
```

Comment:
You new arrivals image is stretching. You can use a better image as the card better suits a long image instead of a wide image. Or you can use object fit cover/contain

Reply:
Oh thanks for that will sure see to it 👍